### PR TITLE
[Pkg] Do not break Mobile Identity Management by default

### LINKED ISSUE #327: Removing com.android.stk and com.google.android.setupwizard breaks mobile identity management
**Your phone**: irrelevant

**Packages documentation to update:**
```
com.android.stk
com.google.android.setupwizard
```

### com.android.stk
**Removal**: Recommended :arrow_right: Advanced

### com.google.android.setupwizard
**Removal**: Recommended :arrow_right: Advanced

Note that it might be necessary to update the documentation for `com.android.stk2` as well.

### PR DESCRIPTION
**Pull request by AnonymousWP**
Fixes #327.

This PR updates the `description` and `removal` values of `com.android.stk`, `com.android.stk2` and `com.google.android.setupwizard`.

### CODE REVIEW

**Comment by nixingyang** [comment]
@AnonymousWP
Thanks for preparing the patch.
The descriptions need to be revised. It is not mobile device management but [mobile identity management](https://en.wikipedia.org/wiki/Mobile_identity_management). In Finland, we have [Mobiilivarmenne](https://mobiilivarmenne.fi/eng/), and removing those packages breaks its functionality.

**Comment by AnonymousWP** [comment]
> @AnonymousWP Thanks for preparing the patch. The descriptions need to be revised. It is not mobile device management but [mobile identity management](https://en.wikipedia.org/wiki/Mobile_identity_management). In Finland, we have [Mobiilivarmenne](https://mobiilivarmenne.fi/eng/), and removing those packages breaks its functionality.

Oops, I automatically read it as Mobile Device Management, my bad. Fixed in a new commit.

**Comment by nixingyang** [comment]
@0x192 Could you take a look into this matter? This merge request looks good to me.

**Comment by 0x192** [comment]
Thanks both of you for your help!

What the heck does `com.google.android.setupwizard` have to do with MIM? So weird.

@nixingyang 
Are 100% sure on this one?

**Comment by nixingyang** [comment]
@0x192 Yes. I am 100% certain that these two packages are required to get [Mobiilivarmenne](https://mobiilivarmenne.fi/eng/) working properly.
Without `com.google.android.setupwizard`, a user will not receive the authorization request sent from the operator. Thus, the functionality breaks.

**Comment by 0x192** [comment]
Ok. Thanks for the confirmation!

File: resources/assets/uad_lists.json [hunk 1]
```diff
@@ -19136,11 +19172,11 @@
   {
     "id": "com.google.android.setupwizard",
     "list": "Google",
-    "description": "Android Setup\nThe new/factory reset device basic configuration setup guides you through the basics of setting up your device.",
+    "description": "Android Setup\nThe new/factory reset device basic configuration setup guides you through the basics of setting up your device.\nWARNING: Oddly enough, disabling/uninstalling this package will break mobile identity management which could be used by apps (for example your Bank) to authenticate you. See https://en.wikipedia.org/wiki/Mobile_identity_management"",
```
Comment (YoranSys) [inline resources/assets/uad_lists.json:19175]: Description break the json, it's end with ""

**Pull request merged**